oxidation/libparsec/crates/crypto/src/sodiumoxide/sequester.rs

```diff
@@ -502,6 +508,26 @@ fn test_pub_signature_verification() {
     assert!(verification.is_ok());
 }
 
+#[test]
+fn test_key_equality() {
+    use hex_literal::hex;
+
+    let pub_key = SequesterPublicKeyDer::try_from(
+        &hex!(
+            "30819f300d06092a864886f70d010101050003818d0030818902818100a0b7653b73670b364cd397"
+            "27a3843502d1ba0402e38f197d4b6ab2ea31da8b23cfef82fdc7bf723d0ae6093767aa92274dde3a"
+            "b0f959f0624d4d5f4c7601a561a646e2f323e60f69e52f53158654f0a592ba0f19cebb3f7b2cbc6b"
+            "7920670c6fe1b23c13a77597dc50d34dc1c454c25ee3348cfae7e089bcf50a266b4033f5f5020301"
+            "0001"
+        )[..],
+    )
+    .unwrap();
+
+    let pub_key_two = pub_key.clone();
+
+    assert!(pub_key == pub_key_two);
```

Comment:
`assert_eq!` is preferable while `rust` doesn't update his `assert!` macro

Reply:
fixed in 6e4b2a88d0d3d91211e8b6719f071ad3d3851797